src/Text/Pandoc/Writers/Markdown/Inline.hs

```diff
@@ -557,6 +641,8 @@ inlineToMarkdown opts lnk@(Link attr@(ident,classes,kvs) txt (src, tit)) = do
     PlainText
       | useAuto -> return $ literal srcSuffix
       | otherwise -> return linktext
+    Markua -> return $ "[" <> linktext <> "](" <> (literal src) <> ")" <> 
+        attrsToMarkua (addKeyValueToAttr attr ("title", tit))
```

Comment:
Do you want to do this even when `tit` is empty?

Reply:
We changed it accordingly